odoo/service/model.py

```diff
@@ -43,7 +43,10 @@ def get_public_method(model: BaseModel, name: str):
     Accessible methods are public (in sense that python defined it:
     not prefixed with "_") and are not decorated with `@api.private`.
     """
-    assert isinstance(model, BaseModel)
```

Comment:
we achieve (almost the same) assertion with the f-string, it is gonna fail for an attribute error `"..." has no "_name"`